server/engine/telemetry.go

```diff
@@ -41,29 +57,73 @@ func StoreMetricFromMainOutputs(db *gorm.DB) {
 	db.Where("module_name = ?", "").Find(&mainOutput)
 
 	if loc, exists := mainOutput.Values["location"]; exists {
-		model.SetMetric(db, model.Region, loc.(map[string]interface{})["value"].(string))
+		model.SetMetric(db, model.Region, loc.(map[string]interface{})["value"].(string), "")
 	} else {
 		log.Error("Location of deployment is missing : will not be included in telemetry")
 	}
 	if access, exists := mainOutput.Values["access"]; exists {
-		model.SetMetric(db, model.AccessType, access.(map[string]interface{})["value"].(string))
+		model.SetMetric(db, model.AccessType, access.(map[string]interface{})["value"].(string), "")
 	} else {
 		log.Error("Access Type of deployment is missing : will not be included in telemetry")
 	}
 }
 
+func StoreRetriesPerStep(db *gorm.DB, step model.Step) {
```

Comment:
Does this function need to be exported(start with capital letter)? The only use is in `StoreMetricsPerStep()`.